src/core.jl

```diff
@@ -98,18 +74,19 @@ end
 function trainGigaSOM(som::Som, dInfo::LoadedDataInfo;
                       kernelFun::Function = gaussianKernel,
                       metric = Euclidean(),
+                      somDistFun = distMatrix(Chebyshev()),
                       knnTreeFun = BruteTree,
-                      rStart = 0.0, rFinal=0.1, radiusFun=linearRadius,
-                      epochs = 10)
+                      rStart = 0.0, rFinal=0.1, radiusFun=expRadius(-5.0),
+                      epochs = 20)
 
     # set the default radius
     if rStart == 0.0
-        rStart = √(som.xdim^2 + som.ydim^2) / 2
+        rStart = (som.xdim + som.ydim) / 2
```

Comment:
This is a big change. Still correct?

Reply:
Yes, this is the result of parameter tuning (also, the new simpler form is much closer to FlowSOM).

Reply:
ok, makes sense

Reply:
Ok, just wanted to check  👍